chapter_01_domain_model.asciidoc

```diff
@@ -2,7 +2,7 @@
 == Domain modelling
 
 In the prologue, we used the term "business logic layer" to describe the
-central layer of a three-layered architecture. For the rest of the book we're
+central layer of a three-layered architecture. For the rest of the book, we're
```

Comment:
not sure about this one

Reply:
I don't think the comma should be there :thinking: